app/user.go

```diff
@@ -0,0 +1,597 @@
+package app
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/gogo/protobuf/proto"
+	"github.com/temporalio/tcld/protogen/api/auth/v1"
+	"github.com/temporalio/tcld/protogen/api/authservice/v1"
+	"github.com/urfave/cli/v2"
+	"strings"
+)
+
+const (
+	userIDFlagName              = "user-id"
+	userEmailFlagName           = "user-email"
+	accountRoleFlagName         = "account-role"
+	namespacePermissionFlagName = "namespace-permission"
```

Comment:
I think we added a --user-namespace-permission flag for namespaces? Do we want to try and consolidate?